protected/components/TargetCheckManager.php

```diff
@@ -1,16 +1,19 @@
 <?php
+
 /**
  * Class TargetCheckManager
  */
-class TargetCheckManager {
+class TargetCheckManager
```

Comment:
There is no need to change code style in this commit - please revert these changes back. The code should conform to our requirements specified in https://stellarbit.atlassian.net/wiki/spaces/STELLARBIT/pages/5439490/PHP+Code+Style+Guideline, as for this branch - the opening curly brace should always be placed on the same line, as the preceding statement (class XXX {, if (..) {, etc).